test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java

```diff
@@ -163,6 +153,59 @@ private V1Pod getTestPod(String className, String methodName, String podName) {
         return pod;
     }
 
+    private static String getArgs() {
```

Comment:
I'm not sure why this hasn't been a problem for other people, or it  has just gone unnoticed, but this makes sure that all `JAVA_OPTS` are properly forwarded to the individual system test job pods. The problem I was running into was due to this, as the `bookkeeperImageVersion` (among others) were not being forwarded to the test pod and caused downstream effects. 

All the options are likely not strictly used, but it would be nice not to worry about this issue going forward.